scheduler/src/cook/mesos/api.clj

```diff
@@ -589,27 +600,28 @@
   "Sets the number of retries for the given job-id to the given retries value,
   if retries is parseable as a long integer and the current user is
   authorized to modify that job."
-  [conn fid request job-id retries]
+  [conn auth-fn fid request job-id requested-retries]
   (let [user (:authorization/user request)
         uuid (maybe-uuid job-id)
         job  (fetch-job-map (db conn)
                             fid
                             job-id) 
         authorized?  (and job
-                          (is-authorized? user :update job))
-        retries (try (Long/parseLong retries)
-                     (catch Throwable t
-                       (log/warn "[set-retries!] User" user
-                                 " submitted an unparseable 'retries' value: " retries
-                                 "Aborting request.")
-                       nil))]
+                          (auth-fn user :update job))
+        retries      (try (Long/parseLong requested-retries)
```

Comment:
Oh. Yes. That is so that the second form on that line aligns visually with the second form on the previous line -- with column alignment, it's easier to scan through the code quickly. The eye can follow the vertical lines to find the place where the second form begins. Without that column alignment, it's necessary to scan and parse the entire line to figure out where the second form begins.

https://github.com/clojure-emacs/clojure-mode#vertical-alignment